client/src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Do we still need this line?

Reply:
yes, tried deleting the line and the styling got messed up without it

Reply:
Nvm, it looks like this line is needed 😅, nice job!